common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -37,6 +37,9 @@ trait Logging {
    * @param message Message to write to the log
    */
   def debug(from: AnyRef, message: String)(implicit id: TransactionId = TransactionId.unknown) = {
+    if (id.meta.extraLogging) {
+      emit(InfoLevel, id, from, message)
+    }
     emit(DebugLevel, id, from, message)
```

Comment:
May be put the DebugLevel log in else to avoid duplicate logs

Reply:
Good catch